Dangerfile

```diff
@@ -10,16 +12,30 @@ def gemfile_lock_version
   gemfile_lock.scan(/fastlane-plugin-wpmreleasetoolkit \((\d+.\d+.\d+)\)/).last.first
 end
 
+github.dismiss_out_of_range_messages
+
+# `files: []` forces rubocop to scan all files, not just the ones modified in the PR
+rubocop.lint(files: [], force_exclusion: true, inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
+
+manifest_pr_checker.check_gemfile_lock_updated
+
+# skip remaining checks if we're during the release process
+if github.pr_labels.include?('Releases')
+  message('This PR has the `Releases` label: some checks will be skipped.')
+  return
+end
+
 # Before checking the version, get rid of any change that `bundle install`
 # might have done.
 `git checkout Gemfile.lock &> /dev/null`
```

Comment:
❓  Does this still make sense given that we have `check_gemfile_lock_updated`? In other words, if `bundle install` ends up changing the `Gemfile.lock` for some reason(†), wouldn't `check_gemfile_lock_updated` already error and fail at that point, meaning that we'll only get to that point if `Gemfile.lock` has not been modified by `bundle install` in the first place?

(†) Though maybe there are cases where the `Gemfile.lock` would be modified by `bundle install` that are not related to it not being up-to-date with `Gemfile`, but for other reasons… I'm especially thinking about the potential addition of a `PLATFORMS` entry…? 🤔

Reply:
> Does this still make sense given that we have check_gemfile_lock_updated? 
My first thought was: this won't hurt, so I just kept it. But at the same time, that being the case, it is also noise, so looking at it again, I'm now thinking of removing it 🤔 

>  I'm especially thinking about the potential addition of a PLATFORMS entry…? 
Good point, I think this can happen.